packages/trader/src/App/Containers/RealAccountSignup/personal-details.jsx

```diff
@@ -224,7 +224,7 @@ class PersonalDetails extends Component {
                                             <DateOfBirth
                                                 name='date_of_birth'
                                                 label={localize('Date of birth*')}
-                                                placeholder={localize('1999-07-01')}
+                                                placeholder={localize('13-01-1999')}
```

Comment:
I think it's fine to keep `01-07-1999` placeholder since it's not something that user types, they select it from the date picker tool so there's no confusion in which one is month and which one is day, unless UI team asked us to change the values?